lib/DDG/Spice/Equaldex.pm

```diff
@@ -0,0 +1,39 @@
+package DDG::Spice::Equaldex;
+# ABSTRACT: LGBT rights by region
+
+use DDG::Spice;
+use Locale::Country; 
+
+spice is_cached => 1;
+
+name "Equaldex";
+source "equaldex";
+icon_url "/i/equaldex.com.ico";
+description "Show LGBT rights by region";
+primary_example_queries "lgbt china";
+secondary_example_queries "lesbian rights cyprus", "gay rights china", "bisexual rights australia", "transgender rights spain";
+category "location_aware";
+topics "special_interest";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Equaldex.pm";
+attribution github => ["MrChrisW", "Chris Wilson"],
+            web => ["http://chrisjwilson.com", "Chris Wilson"];
+
+spice to => 'http://equaldex.com/api/region?format=json&region=$1&callback={{callback}}';
+
+my $triggerRe = qr/^(lgbt|lesbian|gay|bisexual|transgender)\s?(rights?|laws?)\s?(in)?/;
+triggers query_lc => $triggerRe;
```

Comment:
Let's use trigger words here and then check the query against the regex in the handler